templates/base.html

```diff
@@ -54,12 +57,27 @@
         </div>
     </div>
 </header>
+<a class="btn btn-info btn-fab btn-raised mdi-communication-forum"
+   data-target="#modal-post-create"
+   data-toggle="modal"
+   style="left: 50% !important;color: #D7F1FD; margin-top: -125px;">
+</a>
 
 <main class="container">
     {% block content %}
     {% endblock %}
 </main>
 
+<div class="modal fade" id="modal-post-create" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-body">
+                {% include 'posts/_post_create.html' %}
+            </div>
+        </div>
+    </div>
+</div>
+
```

Comment:
This whole modal thing should be in a different PR because I won't merge it. I clashes with the UI built into search.